portifolio.html

```diff
@@ -139,5 +139,36 @@ <h3>Multi-Post Stories</h3>
         </div>
     </div>
 </section>
+<section class="contact-form">
+    <h1>
+        CONTACT US
+    </h1>
+    <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+    </p>
+    <form action="#">
+        <input id="name" name="Name" class="text" type="text" placeholder="Full Name" maxlength="30" required>
+        <input id="email" name="email" class="text" type="email"  placeholder="kgeorgewilliam60@gmail" required>
```

Comment:
- [ ] As users will preferably have/ use different email addresses unlike the one you have for your email placeholder, it will be a good idea to have the email placeholder similar to _Email Address_